source/boot_security.c

```diff
@@ -1,11 +1,37 @@
 #include "boot_security.h"
+#include "simulation.h"
+
+/* Remove the include */
+#include <windows.h> // only runs on Windows
+
+#include "mbedtls\aes.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include "hal_log.h"
+
+#define EXAMPLE_AES_KEY_LEN_IN_BITS		256
+#define EXAMPLE_AESKEY_LEN_IN_BYTES		(EXAMPLE_AES_KEY_LEN_IN_BITS / 8)
+#define AES_IV_LENGTH					(16)
+#define LOG_PRINTF hal_log_printf
+mbedtls_aes_context aes;
```

Comment:
That's fine but could you move this variable into boot_decrypt_upgrade_package() function. 
This is because only boot_decrypt_upgrade_package() function uses this variable and there is no need to make it a global variable.